src/assets/Timecard.scss

```diff
@@ -3,3 +3,61 @@
     margin-top: 20px;
```

Comment:
Could you provide screenshots in the description. Demonstrating the changes made for this PR

Reply:
Description updated with before and after screenshots